elections/ELECTIONS.md

```diff
@@ -0,0 +1,17 @@
+# TSC Elections Process
+## Guidelines
+1. Avoid scheduling around major holidays.  Use best judgement to determine where TSC members are located and which dates they may be unavailable.
+2. Announce the election in the [prestodb/presto main repo README](https://github.com/prestodb/presto)
+
+## Process
+1. The facilitator will announce the following dates:
+    1. D: the date that nominations will be open; a PR will be opened updating the PrestoDB README announcing the election; and an issue will be opened announcing the election in the TSC repo.
+    2. D+7: the date that nominations will be closed; a comment will be made in the issue announcing current candidates and any invalid candidates.
```

Comment:
probably say `; a comment will be made in the issue announcing candidates with valid and invalid nominations`?
Should we notify invalid nominations to candidates over email as well?

Reply:
SG